NeoML/include/NeoML/Dnn/Layers/TiedEmbeddingsLayer.h

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 
```

Comment:
Fix for all changed files
from
/* Copyright © 2017-2020 ABBYY Production LLC
to
/* Copyright © 2017-2024 ABBYY